purchase_deposit/wizard/purchase_make_invoice_advance.py

```diff
@@ -154,8 +154,22 @@ def _prepare_advance_purchase_line(self, order, product, tax_ids, amount):
             "taxes_id": [(6, 0, tax_ids)],
             "date_planned": datetime.today().strftime(DEFAULT_SERVER_DATETIME_FORMAT),
             "is_deposit": True,
+            "sequence": order.order_line and order.order_line[-1].sequence + 1 or 10,
```

Comment:
I guess 10 can be set as the value of the sequence because that's the default value of the field in its definition. However, that value could be replaced by another one, so I suggest using the following line of code to get the actual default value:
`seq = self.env['purchase.order.line'].default_get(['sequence']).get('sequence')`